### DISCUSSION TITLE
Pow node?

### DISCUSSION THREAD

**Discussion by treefrogman** [maximecb/noisecraft#95] [Ideas]
Granted, this was a lot of fun to figure out: https://noisecraft.app/607
However, it could be pretty handy to be able to do that more compactly—even if/when we do get a quantizer node.

**Comment by maximecb** [comment]
Hehe pretty fun little note generator you got there.

For the pow node, you'd want just a base and exponent input, base/exp?

One thing is that we can't ever produce NaN values (because they propagate everywhere), so we'd need to figure out all the values that produce corner cases and always return zero or something like that. Like I'm assuming we don't want to support a base less than zero, or an exponent less than zero. Might also be able to just use the JS Math.pow and return 0 whenever it produces NaN.

**Reply by treefrogman** [reply]
Haha yes! What you said at the end there was my first thought. It would be simpler code than checking that the inputs are in range. And who knows, the erratic behavior below zero may be just the ticket for some creative uses.
